packages/react-instantsearch-hooks/src/lib/__tests__/useSearchResults.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { AlgoliaSearchHelper, SearchResults } from 'algoliasearch-helper';
+
+import { createInstantSearchTestWrapper } from '../../../../../test/utils';
+import { useSearchResults } from '../useSearchResults';
+
+describe('useSearchResults', () => {
+  test('returns the connector render state', async () => {
+    const wrapper = createInstantSearchTestWrapper();
+    const { result, waitForNextUpdate } = renderHook(() => useSearchResults(), {
+      wrapper,
+    });
+
+    // Initial render state from manual `getWidgetRenderState`
+    expect(result.current).toEqual({
+      results: expect.any(SearchResults),
+      scopedResults: [
+        expect.objectContaining({
+          helper: expect.any(AlgoliaSearchHelper),
+          indexId: 'indexName',
+          results: expect.any(SearchResults),
+        }),
+      ],
+    });
+
```

Comment:
We can also add this check, because we'll use this to detect the "no results" scenario.

```suggestion
    expect(result.current.results.__isArtificial).toEqual(true);

```